client/galaxy/scripts/apps/analysis.js

```diff
@@ -183,6 +195,23 @@ window.app = function app( options, bootstrapped ){
             this.page.$( '#galaxy_main' ).prop( 'src', url );
         },
 
+        /** load workflow by its url in run mode */
+        _loadWorkflow: function() {
+            var self = this;
+            self.page.$( '#galaxy-main' ).hide();
+            self.page.$( "#center-panel" ).show();
+            Utils.get({
+                url : Galaxy.root + 'workflow/run_workflow?id=' + Utils.getQueryString( 'id' ),
+                success : function( response ) {
+                    var form = new ToolFormComposite.View( response );
+                    self.page.$( "#center-panel" ).empty().append( form.$el )
```

Comment:
It works. I am not sure how it would when we need to show an error in the async call.